### PR TITLE
calculate velocity constant and max vel from parameters

### PR DESCRIPTION
**Pull request by Kitajima-robotis**
Turtlebot3 has many people who change the wheel radius by customizing the hardware. 

This PR calculates the `rate constant` and `min/max velocity` from the parameters (such as `WHEEL_RADIUS`), making it easier and more intuitive to adapt the software to the customized hardware.

### CODE REVIEW

**Comment by routiful** [comment]
Thanks :)

When calculate MAX_LINEAR_VELOCITY, what is the meaning of number 61(burger) and 77(waffle) ?

**Comment by Kitajima-robotis** [comment]
Both values are the Dyanamixel 's rpm value.
Burger uses rpm at no load speed of Dynamixel XL430-W250. (at 12 v)
Also, Waffle Pi uses rpm at no load speed of Dynamixel XM430-W210. (at 12 v)

The value is based on the e-manual's no load speed.
[e-Manual v1.31.30 XL430-W250](http://support.robotis.com/en/product/actuator/dynamixel_x/xl_series/xl430-w250.htm)
[e-Manual v1.31.30 XM430-W210](http://support.robotis.com/en/product/actuator/dynamixel_x/xm_series/xm430-w210.htm)

File: arduino/opencr_arduino/opencr/libraries/turtlebot3/examples/turtlebot3_burger/turtlebot3_core/turtlebot3_burger.h [hunk 1]
```diff
@@ -28,8 +28,8 @@
 #define ENCODER_MIN                      -2147483648     // raw
 #define ENCODER_MAX                      2147483648      // raw
 
-#define MAX_LINEAR_VELOCITY              0.22   // m/s   (BURGER : 0.22, WAFFLE : 0.25)
-#define MAX_ANGULAR_VELOCITY             2.84   // rad/s (BURGER : 2.84, WAFFLE : 1.82)
+#define MAX_LINEAR_VELOCITY              (WHEEL_RADIUS * 2 * 3.14159265359 * 61 / 60) // m/s  (BURGER : 61, WAFFLE : 77)
```
Comment (routiful) [inline arduino/opencr_arduino/opencr/libraries/turtlebot3/examples/turtlebot3_burger/turtlebot3_core/turtlebot3_burger.h]: Unit is necessary after number.

File: arduino/opencr_arduino/opencr/libraries/turtlebot3/examples/turtlebot3_with_open_manipulator/turtlebot3_with_open_manipulator_core/turtlebot3_with_open_manipulator.h [hunk 2]
```diff
@@ -32,8 +32,10 @@
 #define PALM_NUM                         2  // grip_joint + grip_joint_sub
 #define GRIP_NUM                         1    
 
-#define MAX_LINEAR_VELOCITY              0.25   // m/s   (BURGER : 0.22, WAFFLE : 0.25)
-#define MAX_ANGULAR_VELOCITY             1.82   // rad/s (BURGER : 2.84, WAFFLE : 1.82)
+#define MAX_LINEAR_VELOCITY              (WHEEL_RADIUS * 2 * 3.14159265359 * 61 / 60) // m/s  (BURGER : 61, WAFFLE : 77)
```
Comment (routiful) [inline arduino/opencr_arduino/opencr/libraries/turtlebot3/examples/turtlebot3_with_open_manipulator/turtlebot3_with_open_manipulator_core/turtlebot3_with_open_manipulator.h]: OpenManipulator is only assembled with Waffle.

**Comment by Kitajima-robotis** [comment]
Thank you for review.

I fixed the following two.
・Added a unit after the number.
・Changed OpenManipulator's number.(61[rpm] -> 77[rpm])

**Comment by routiful** [comment]
Great! Thank you for contribution.
I will test this code and merge master branch :)

**Pull request merged**